pilot/cmd/pilot-agent/main.go

```diff
@@ -416,6 +417,8 @@ func init() {
 		"HTTP Port on which to serve pilot agent status. If zero, agent status will not be provided.")
 	proxyCmd.PersistentFlags().StringSliceVar(&applicationPorts, "applicationPorts", []string{},
 		"Ports exposed by the application. Used to determine that Envoy is configured and ready to receive traffic.")
+	proxyCmd.PersistentFlags().BoolVar(&waitForCerts, "waitForCerts", false,
```

Comment:
@mandarjog I wonder if we should be using environment variables for all new pilot-agent features to avoid the upgrade issues.  WDYT?